src/Symfony/Component/Validator/Constraints/AbstractComparison.php

```diff
@@ -24,15 +24,16 @@
 {
     public $message;
     public $value;
+    public $propertyPath;
 
     /**
      * {@inheritdoc}
      */
     public function __construct($options = null)
     {
-        if (is_array($options) && !isset($options['value'])) {
+        if (is_array($options) && !isset($options['value']) && !isset($options['propertyPath'])) {
```

Comment:
We should also error if both the `value` and the `propertyPath` option are given.